dracut-initramfs-restore.sh

```diff
@@ -13,6 +13,7 @@ SKIP="$dracutbasedir/skipcpio"
 [[ -x $SKIP ]] || SKIP=cat
 
 [[ -f /etc/machine-id ]] && read MACHINE_ID < /etc/machine-id
+[[ -f "/lib/modules/${KERNEL_VERSION}/pkgbase" ]] && read PKGBASE < "/lib/modules/${KERNEL_VERSION}/pkgbase"
```

Comment:
This isn't gonna work after a kernel update as `/lib/modules/$(uname -r)/pkgbase` won't exist

Reply:
right, good catch; so I guess there is no simple way to get it after an update.

Reply:
pkgbase should always exist on the official Arch kernels

Reply:
indeed, but after an update, the parent directory (`unamre -r`) does not exist anymore.